src/services/completions.ts

```diff
@@ -2464,6 +2466,29 @@ function getCompletionEntryCodeActionsAndSourceDisplay(
         return { codeActions: [codeAction], sourceDisplay: undefined };
     }
 
+    if (contextToken && isObjectLiteralExpression(contextToken.parent.parent) && previousToken?.kind !== SyntaxKind.ColonToken) {
+        let changes: FileTextChanges[];
+        if (getLineAndCharacterOfPosition(sourceFile, contextToken.getEnd()).line !== getLineAndCharacterOfPosition(sourceFile, position).line) {
+            changes = textChanges.ChangeTracker.with(
+                { host, formatContext, preferences },
+                tracker=>tracker.replacePropertyAssignment(sourceFile, contextToken.parent as PropertyAssignment ,contextToken.parent as PropertyAssignment));
+        }
+        else {
+            changes = textChanges.ChangeTracker.with(
+                { host, formatContext, preferences },
+                tracker=>tracker.replacePropertyAssignmentOnSameLine(sourceFile, contextToken.parent as PropertyAssignment ,contextToken.parent as PropertyAssignment));
+        }
+        if (changes) {
+            return {
+                sourceDisplay: undefined,
+                codeActions: [{
+                    changes,
+                    description: diagnosticToString([Diagnostics.Includes_imports_of_types_referenced_by_0, name]),

```

Comment:
I think you copied and pasted this and forgot to change the message—it doesn’t apply to this fix. You’ll need to make a new diagnostic message to reference here.